editor/js/Sidebar.Settings.Shortcuts.js

```diff
@@ -145,6 +145,16 @@ Sidebar.Settings.Shortcuts = function ( editor ) {
 
 				break;
 
+			case config.getKey( 'settings/shortcuts/focus' ):
+
+				if ( editor.selected !== null ) {
+
+					editor.focusById( editor.selected.id );
```

Comment:
How about `editor.focus( editor.selected );` instead?

Reply:
Yeah, that's a lot cleaner! I just changed it, and that fixed the bug that occurred when trying to focus on the camera as well, so that's nice.